schemas/Config.json

```diff
@@ -0,0 +1,332 @@
+{
+  "definitions": {
```

Comment:
>"definitions": { [](start = 2, length = 16)

Where does this file come from?

Reply:
generated by CreateJsonSchema

Reply:
I mean I can see like LandingData.json generated from LandingData.cs, was this generated from Config.cs?

---
In reply to: [210524771](https://github.com/dotnet/docfx/pull/3216#discussion_r210524771) [](ancestors = 210524771)

Reply:
the idea is that if you make any changes that changes JSON schema, CI can detect that and force you regenerate a new JSON schema by running build.ps1 locally. This ensures JSON schema is always up to date with code changes.

Reply:
yes, it is hard coded as `CreateJsonSchema(typeof(Config))`